mithril-common/src/entities/cardano_stake_distribution.rs

```diff
@@ -0,0 +1,175 @@
+use serde::{Deserialize, Serialize};
+use sha2::{Digest, Sha256};
+
+use crate::signable_builder::Artifact;
+
+use super::{Epoch, StakeDistribution};
+
+/// Cardano Stake Distribution
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct CardanoStakeDistribution {
+    /// Unique hash of the Cardano Stake Distribution
+    pub hash: String,
+
+    /// Epoch at which the Cardano Stake Distribution is created
+    pub epoch: Epoch,
+
+    /// List of pools with their stakes of the Cardano Stake Distribution
```

Comment:
Could you formulate that in a clearer way?

Reply:
Done ✅
https://github.com/input-output-hk/mithril/pull/1847/files#diff-8bf6e1f4a93a65fb277085ac8e9cc0aff8f745fc1ff918f0b67debf58e38f2faR17